pb-crm-api/src/db/entities/phone.entity.ts

```diff
@@ -1,22 +1,33 @@
 import {Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn} from "typeorm";
 import {ContactEntity} from "./contact.entity";
-import {PhoneModel} from '@hiddentemple/api-interfaces';
-import {IsOptional, MaxLength, IsPhoneNumber} from "class-validator"
+import {IsDefined, IsNumberString, IsPhoneNumber, Length, ValidateNested} from "class-validator"
+import {PhoneCategoryEntity} from "./phone-category.entity";
+import {PhoneModel} from "@hiddentemple/api-interfaces";
+import {Type} from "class-transformer";
 
 @Entity('phones')
 export class PhoneEntity implements PhoneModel {
 
     @PrimaryGeneratedColumn('uuid')
     id: string
 
-    @Column('numeric', { nullable: false })
+    @Column('numeric', {nullable: false})
+    @IsDefined()
+    @IsNumberString( { no_symbols: true })
+    @Length(10, 10, {message: 'Phone number must be exactly 10 digits'})
     @IsPhoneNumber('US')
-    number: number
+    phoneNumber: string
```

Comment:
does the numeric validator cause issues with the type being string?

Reply:
Great question, I'm blanking on why I changed this. It was one of the validators, can't remember which. We could add a class transformer that would allow the field to be set as a string, but always persisted and stored in memory as a number. Idk, do you think it should be a number or a string?

Reply:
I think it should be string, was just curious if the validator played ok with the string type.

Reply:
Wasn't one of the issues with phone number last sprint the fact that it was being converted from a `string` to a `number`?

Reply:
I think that was happening in the factory nick. Not sure tho. Either way, when the changes to the controller get merged, I'll be sure to validate this isn't an issue.